src/search/_Search.jsx

```diff
@@ -102,6 +102,13 @@ const Search = ({ fetchSearchItems, onItemSelected }) => {
   cacheSearchItems(fetchSearchItems, state, setState);
   const { searchItems, searchString, results } = state;
   const disclosureManager = React.useContext(DisclosureManagerContext);
+
+  let searchRef = useRef(null);
+
+  useEffect(() => {
+    searchRef.focus();
+  });
```

Comment:
It might be safe as is, but you could include the memo param here to ensure it only gets executed on mount.
```suggestion
  }, []);
```